statsmodels/tsa/tests/test_stattools.py

```diff
@@ -11,12 +11,11 @@
 from statsmodels.compat.numpy import recarray_select
 from statsmodels.compat.python import lrange
 from statsmodels.datasets import macrodata, sunspots
-from statsmodels.tools.sm_exceptions import ColinearityWarning
-from statsmodels.tools.sm_exceptions import MissingDataError
+from statsmodels.tools.sm_exceptions import CollinearityWarning, MissingDataError
```

Comment:
this line looks too long now, use parenthesis

Reply:
Lots of places seem to use 100, which is within the pep spec.  I definitely prefer 100.